### PR TITLE
Quick fix nexus5 point/spot light issues

### PR DESCRIPTION
**Pull request by bhouston**
This PR fixes the issue with more than one point or spot light not working on Nexus 5 devices, addressing long standing issues #9100 #9871.

To fix this I removed the testLightInRange function which was added in this commit: https://github.com/mrdoob/three.js/commit/3adb436bad83f879592b299b6155f3c9dc3a746f 

I doubt this is really the best fix.  I know that the testLightInRange function does hvae issues in that it doesn't look at decayExponent in the same way that it is used in punctualLightIntensityToIrradianceFactor.  Basically testLightInRange is a more stringent of a test.

To be specific in punctualLightIntensityToIrradianceFactor, decayExponent must be > 0 for the cutoffDistance to matter.  But testLightInRange always uses cutoffDistance whether or not the light is decaying.

I think there is something else going on at the same time. I will investigate a bit more today to see if there is a better fix -- I suspect that an array of uniforms (either cutoffDistance or decayExponent) is not being initialized properly, otherwise why would this only affect the lights after the first....

### CODE REVIEW

**Comment by bhouston** [comment]
This fix was tested only on http://threejs.org/examples/#webgl_lights_pointlights

**Comment by bhouston** [comment]
I've tried to figure out the real underlying issue in testLightInRange and I can not figure it out.  The parameters seem correct, the check just seems to fail on N > 0 for lights no matter what I set, unless I set it to return true.

I'm going to move on if people find htis PR sufficient.

**Comment by mrdoob** [comment]
> To fix this I removed the testLightInRange function which was added in this commit: https://github.com/mrdoob/three.js/commit/3adb436bad83f879592b299b6155f3c9dc3a746f 

Eek! Sorry about that 😞

File: src/renderers/shaders/ShaderChunk/lights_pars.glsl [hunk 1]
```diff
@@ -63,19 +63,9 @@ vec3 getAmbientLightIrradiance( const in vec3 ambientLightColor ) {
...
-
-		} else {
-
-			directLight.color = vec3( 0.0 );
-			directLight.visible = false;
-
-		}
+		directLight.color = pointLight.color;
+		directLight.color *= punctualLightIntensityToIrradianceFactor( lightDistance, pointLight.distance, pointLight.decay );
+		directLight.visible = ( directLight.color != vec3( 0.0, 0.0, 0.0 ) );
```
Comment (mrdoob) [inline src/renderers/shaders/ShaderChunk/lights_pars.glsl]: Would there be any benefit of adding `#define COLOR_BLACK vec3( 0.0 )` in `common.glsl` for this?

Comment (bhouston) [inline src/renderers/shaders/ShaderChunk/lights_pars.glsl]: The issue with COLOR_BLACK is that color can be RGB or RGBA, so we would need to defines RGB_BLACK and RGBA_BLACK but really it is only saving a few characters.  I did replace my usage of vec3( 0.0, 0.0, 0.0 ) with vec3( 0.0 ) which is what I should have done in the first place.

**Comment by bhouston** [comment]
This is ready for merging.  I tested all examples and they seem good.

**Comment by mrdoob** [comment]
Many many thanks!

**Pull request merged**